static/styles/popovers.css

```diff
@@ -66,6 +66,10 @@
         display: none;
         margin-right: 10px;
 
+        @media (max-width: 485px) {
```

Comment:
How did you pick 485px as the cutoff for this?

Reply:
Hey @timabbott 

I progressively went down on screen size (primary on Chrome dev tools, but tested on other browsers as well) and the first sign of the popover overflowing happened at 485px. Happy to change that if you'd like.